rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initalize(date)
+    @date = date
+    @person = person
+    person.rentals << self
+    @book = book
+    book.rental << self
```

Comment:
- [ ] Here should be `book.rentals << self` not `book.rental`

Reply:
This issue is well resolved! ✔️